docusaurus/docs/core-concepts/config-store/overview.md

```diff
@@ -1,35 +1,56 @@
-# Configuration
-Ziti has the capability to define a configuration schema for an application and provide service configuration for those applications which need additional metadata. 
+# Configs
```

Comment:
I personally use the term _Ziti Service Configurations_. I don't especially care what we use here, but I do think the context of _service_ is helpful.